src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/model/ccd/DocumentToRemoveCollection.java

```diff
@@ -0,0 +1,16 @@
+package uk.gov.hmcts.reform.finrem.caseorchestration.model.ccd;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@JsonIgnoreProperties(ignoreUnknown = true)
+@Data
+@Builder
+@AllArgsConstructor
+@NoArgsConstructor
+public class DocumentToRemoveCollection {
+    private DocumentToRemove value;
+}
```

Comment:
As discussed, it's actually ended up being a document to keep collection.  Can discuss.

Reply:
@ptrelease, happy to refactor when it comes to your PR and when we bring in the CCD PR